Bugsnag/KSCrash/Source/KSCrash/Recording/Tools/BSG_KSMach.c

```diff
@@ -65,6 +68,15 @@ uint64_t bsg_ksmachfreeMemory(void) {
 }
 
 uint64_t bsg_ksmachusableMemory(void) {
+#if TARGET_OS_IOS
+    // Crashes on TVOS, not available on MacOS.
+    if (__builtin_available(iOS 13.0, *)) {
+        size_t mem = os_proc_available_memory();
```

Comment:
I think this belongs in `bsg_ksmachfreeMemory`

As the memory footprint of the app increases, the value returned by `os_proc_available_memory` tends towards 0

Reply:
`bsg_ksmachfreeMemory` is a relic from the old KSCrash days before I understood much about how Apple's memory systems work. It should be removed, and places that use it updated to use a different call.

Reply:
The value returned from `bsg_ksmachfreeMemory` is used for `freeMemory` in the crash reports / dashboard, and the value from `bsg_ksmachusableMemory` is shown as `totalMemory`

Reply:
Yes you're right. The existing calculation of free memory is so wrong it threw me off :P

usable memory is also incorrect, but that's for another day.